app/templates/components/gh-image-uploader.hbs

```diff
@@ -19,10 +19,11 @@
                 <div class="description">{{description}}</div>
             {{/gh-file-input}}
         </div>
-
-        <a class="image-url" {{action "switchForm" "url-input"}}>
-            <i class="icon-link"><span class="hidden">URL</span></i>
-        </a>
+        {{#if allowUrlInput}}
```

Comment:
I disabled the blog icon upload form in general settings now, if `fileStorage = false`.